### PR TITLE
[wave2-1073] compare program fixes

### PR DESCRIPTION
**Pull request by mhughes2012**

### CODE REVIEW

File: sites/all/modules/ucalgary/ucgy_fgs/templates/views-view-fields--fgs-program-comparison--block.tpl.php [hunk 1]
```diff
@@ -71,23 +71,23 @@
               <?php if (isset($fields['field_fgs_gpa']) && $gpatext != '') :?>
                    <li><span class="title"><?php echo t('GPA');?></span><br><?php echo $gpatext; ?></li>
               <?php endif; ?>
-              <?php $prerequisitestext = strip_tags($fields['field_fgs_prerequisites']->content); ?>
+              <?php $prerequisitestext = ($fields['field_fgs_prerequisites']->content); ?>
```
Comment (gsowrey) [inline sites/all/modules/ucalgary/ucgy_fgs/templates/views-view-fields--fgs-program-comparison--block.tpl.php]: There should be an isset for this

File: sites/all/modules/ucalgary/ucgy_fgs/templates/views-view-fields--fgs-program-comparison--block.tpl.php [hunk 2]
```diff
@@ -71,23 +71,23 @@
...
+              <?php $completiontext = ($fields['field_fgsif_completion']->content); ?>
               <?php if (isset($fields['field_fgsif_completion']) && $completiontext != '') :?>
                     <li><span class="title"><?php echo t('Completion');?></span><br><?php echo $completiontext; ?></li>
               <?php endif; ?>
               <?php $supervisortext = strip_tags($fields['field_fgsif_supervisor_options']->content); ?>
               <?php if (isset($fields['field_fgsif_supervisor_options']) && $supervisortext != '') :?>
                     <li><span class="title"><?php echo t('Supervisor');?></span><br><?php echo $supervisortext; ?></li>
               <?php endif; ?>
-              <?php $commitmenttext = strip_tags($fields['field_fgsif_commitment']->content); ?>
+              <?php $commitmenttext = ($fields['field_fgsif_commitment']->content); ?>
```
Comment (gsowrey) [inline sites/all/modules/ucalgary/ucgy_fgs/templates/views-view-fields--fgs-program-comparison--block.tpl.php]: There should be an isset for this

File: sites/all/modules/ucalgary/ucgy_fgs/templates/views-view-fields--fgs-program-comparison--block.tpl.php [hunk 3]
```diff
@@ -71,23 +71,23 @@
...
               <?php if (isset($fields['field_fgsif_commitment']) && $commitmenttext != '') :?>
                     <li><span class="title"><?php echo t('Commitment');?></span><br><?php echo $commitmenttext; ?></li>
               <?php endif; ?>
-              <?php $awardstext = strip_tags($fields['field_fgsif_awards_and_funding']->content); ?>
+              <?php $awardstext = ($fields['field_fgsif_awards_and_funding']->content); ?>
```
Comment (gsowrey) [inline sites/all/modules/ucalgary/ucgy_fgs/templates/views-view-fields--fgs-program-comparison--block.tpl.php]: This should have an isset ... maybe two

File: sites/all/modules/ucalgary/ucgy_fgs/includes/rs_filters.inc [hunk 4]
```diff
@@ -73,8 +73,10 @@ $faculties = entity_load_multiple_by_name('faculty');
         <div class="col-sm-12 compare">
           <p>
             <?php
-            // $view->result count set in template
-            if ($count < 1): ?>
+            // check if we're comparing any programs
+            $compare = $_GET['compare'];
```
Comment (mhughes2012) [inline sites/all/modules/ucalgary/ucgy_fgs/includes/rs_filters.inc]: added an isset

File: sites/all/modules/ucalgary/ucgy_fgs/includes/rs_filters.inc [hunk 5]
```diff
@@ -73,8 +73,10 @@ $faculties = entity_load_multiple_by_name('faculty');
...
+
+            if (!$compare): ?>
```
Comment (mhughes2012) [inline sites/all/modules/ucalgary/ucgy_fgs/includes/rs_filters.inc]: updated the if statement to just check to see if the get value is set

File: sites/all/modules/ucalgary/ucgy_fgs/templates/views-view-fields--fgs-program-comparison--block.tpl.php (hunk 1 above)
Comment (mhughes2012) [inline sites/all/modules/ucalgary/ucgy_fgs/templates/views-view-fields--fgs-program-comparison--block.tpl.php]: moved the issets to above the call to the content, avoiding notices

**Pull request merged**